codecs/src/test/scala/org/virtuslab/akkasaferserializer/StandardCodecsSpec.scala

```diff
@@ -1,21 +1,26 @@
 package org.virtuslab.akkasaferserializer
 
-import io.bullet.borer.Cbor
-import org.scalatest.matchers.should
+import io.bullet.borer.Codec
+import io.bullet.borer.derivation.MapBasedCodecs.deriveAllCodecs
 import org.scalatest.wordspec.AnyWordSpecLike
+import org.virtuslab.akkasaferserializer.CodecsData.DateTimeClass
 
+import java.time.OffsetDateTime
 import scala.concurrent.duration.FiniteDuration
 import scala.language.postfixOps
 
-class StandardCodecsSpec extends AnyWordSpecLike with should.Matchers {
+class StandardCodecsSpec extends AnyWordSpecLike with BorerSerializationTestKit {
   import StandardCodecs._
+  implicit val codec: Codec[CodecsData] = deriveAllCodecs
 
   "Defined codes" should {
-    "handle FiniteDuration" in {
-      val test = FiniteDuration(10, "s")
-      val ser = Cbor.encode(test).toByteArray
-      val res = Cbor.decode(ser).to[FiniteDuration].value
-      res should equal(res)
+
+    "serialize FiniteDuration" in {
+      roundTrip(FiniteDuration(10, "s"))
```

Comment:
Nit: maybe `verifyRoundTrip`, to align with `akka.actor.testkit.typed.scaladsl.SerializationTestKit`